frontend/src/modules/admin/services/users.service.ts

```
@@ -0,0 +1,17 @@
+    const tenantId = AuthService.getTenantId();
+
+    const response = await authAxios.get(
+      `/tenant/${tenantId}/user`,
+      {
+        params: query,
+      },
+    );
+
+    return response.data;
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)